tools/workspace-plugin/src/generators/normalize-package-dependencies/index.ts

```diff
@@ -21,6 +21,7 @@ type ProjectIssues = { [projectName: string]: { [depName: string]: string } };
 
 const NORMALIZED_INNER_WORKSPACE_VERSION = '*';
 const NORMALIZED_PRERELEASE_RANGE_VERSION = '>=9.0.0-alpha';
+const NORMALIZED_WEB_COMPONENTS_PRERELEASE_RANGE_VERSION = '>=3.0.0-beta';
```

Comment:
this might not do what one thinks it will do. remove this once https://github.com/microsoft/fluentui/pull/31293 lands